src/services/feed.ts

```diff
@@ -6,6 +6,7 @@ const CONFLUENCE_BASE = 'https://acf-ohs.atlassian.net/wiki/createrssfeed.action
 const FEED_URLS = {
   whatsNew: `${CONFLUENCE_BASE}?types=blogpost&spaces=conf_all&title=Whats+New&labelString=whatsnew&excludedSpaceKeys%3D&sort=modified&maxResults=10&timeSpan=90&showContent=true&confirm=Create+RSS+Feed&os_authType=basic`,
   singleByTag: (tag: string) => `${CONFLUENCE_BASE}?types=page&types=blogpost&spaces=conf_all&title=Tag+${tag}&labelString=${tag}&excludedSpaceKeys%3D&sort=modified&maxResults=1&timeSpan=365&showContent=true&confirm=Create+RSS+Feed&os_authType=basic`,
+  classGuidance: `${CONFLUENCE_BASE}?types=page&spaces=OHSTTA&title=Confluence+RSS+Feed&labelString=ttahub-class-thresholds&excludedSpaceKeys%3D&sort=modified&maxResults=10&timeSpan=5&showContent=true&confirm=Create+RSS+Feed&os_authType=basic`,
```

Comment:
This has no items - also, we can't use the "singleByTag" function and pass in "ttahub-class-threshold"?

Reply:
Maybe you are joining multiple feed entries?

Reply:
Oh weird, that was working just fine last week. Hmm..